test/pylib/manager_client.py

```diff
@@ -19,6 +19,19 @@
 logger = logging.getLogger(__name__)
 
 
+ServerNum = NewType('ServerNum', int)
+IPAddress = NewType('IPAddress', str)
+HostID = NewType('HostID', str)
+
+
+class ServerInfo(NamedTuple):
+    """Server id (UUID) and IP address"""
```

Comment:
"Server id (UUID)"

Reply:
The named tuple has the IP address also

Reply:
But it doesn't have a "Server id (UUID)" <- it's an error